src/index.css

```
@@ -1,64 +1,123 @@
+    --tw-scrollbar-thumb: #bbb; /* Light gray thumb */
+  }
+
+  /* Dark mode colors: black track, dark gray thumb */
+  .dark & {
+    --tw-scrollbar-track: var(--color-neutral-900); /* Black track */
+    --tw-scrollbar-thumb: #555; /* Dark gray thumb */
+  }
+
+  /* Dark mode colors for modals: black track */
+  .dark &.modal {
+    --tw-scrollbar-track: theme(
```

Biome (1.9.4)

[error] Unexpected unknown function: theme

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)